### PR TITLE
Hello, I had made all the required cheanges in the footer, please hav…

### PR DESCRIPTION
**Pull request by AbhishekTiwari23**
…e a look on that.

### All Submissions

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally prior to submission?

### Changes to Core Features

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by AbhishekTiwari23** [comment]
image
I have made all the required footer changes, making it more attractive and eye-catching.
Added social Icons.
I added the Hover effect in each incone.
Heighleted More Info line.

**Comment by AbhishekTiwari23** [comment]
Please make a response.

File: styles.css [hunk 1]
```diff
@@ -661,7 +661,63 @@ p {
...
+}
 .footer p {
-  color: #9c9c9c;
+  color: grey;
   padding-right: 8px;
+  margin: 10px;
+}
+.footer a:hover{
+  color : rgb(196, 24, 179);
+}
```
Comment (Kshitij978) [inline styles.css:675]: Let's don't use this color on GitHub icon as it doesn't fit well with the whole theme of the website. We can just scale on hover without changing the color.

File: styles.css [hunk 2]
```diff
@@ -661,7 +661,63 @@ p {
...
+  margin: 3px;
+}
+
+a i {
+  transition: all 0.3s linear;
+}
+
+a:hover i {
+  transform: scale(1.5);
 }
```
Comment (Kshitij978) [inline styles.css:697]:
```suggestion
.link:hover i {
  transform: scale(1.05);
}
```

**Review by Kshitij978 — CHANGES_REQUESTED** [review]
Hi @AbhishekTiwari23 thanks for the effort. Nice hover effect 👍✨I have suggested few changes to be made before finally merging this PR. 

- Please delete the [.DS_Store](https://github.com/Ashutosh00710/github-readme-activity-graph/pull/151/files#diff-e2eb93a61ffd7877ea5c751abcb3a618e8e2e9a2073a27f66d4114fe10819f86) & [github-readme-activity-graph Submodule](https://github.com/Ashutosh00710/github-readme-activity-graph/pull/151/files#diff-bb6a6a9d2a3f8829c84b5a3d99e120f8334975e802e7096466045bf28e6e9910) from the commit.

- Let's don't use any **social icons** for now as we don't have any socials for this project yet. You can keep your styling only on the **GitHub icon** after removing everything related to social icon and after applying the suggested changes.

**Comment by AbhishekTiwari23** [comment]
> Hi @AbhishekTiwari23 thanks for the effort. Nice hover effect 👍✨I have suggested few changes to be made before finally merging this PR.
> 
> * Please delete the [.DS_Store](https://github.com/Ashutosh00710/github-readme-activity-graph/pull/151/files#diff-e2eb93a61ffd7877ea5c751abcb3a618e8e2e9a2073a27f66d4114fe10819f86) & [github-readme-activity-graph Submodule](https://github.com/Ashutosh00710/github-readme-activity-graph/pull/151/files#diff-bb6a6a9d2a3f8829c84b5a3d99e120f8334975e802e7096466045bf28e6e9910) from the commit.
> * Let's don't use any **social icons** for now as we don't have any socials for this project yet. You can keep your styling only on the **GitHub icon** after removing everything related to social icon and after applying the suggested changes.

Is it fine ?

https://user-images.githubusercontent.com/97469132/194028403-3cab410d-6acc-439c-8649-58125ceb3f5e.mov

**Comment by AbhishekTiwari23** [comment]
@Kshitij978, sir, if you find it relevant, then please let me know so that I can make my first PR

**Comment by AbhishekTiwari23** [comment]
> @AbhishekTiwari23 everything looks great in the screen recording ✨
> 
> Please add these changes too starting from here: [#151 (comment)](https://github.com/Ashutosh00710/github-readme-activity-graph/pull/151#discussion_r987151544) to here: [#151 (comment)](https://github.com/Ashutosh00710/github-readme-activity-graph/pull/151#discussion_r987661842)
> 
> Also remove [this folder](https://github.com/Ashutosh00710/github-readme-activity-graph/pull/151/files#diff-bb6a6a9d2a3f8829c84b5a3d99e120f8334975e802e7096466045bf28e6e9910)
> 
> You can commit after doing all these changes. 👍
> 
> P.S. Let's call each other by name. We both are learners. 😄

Hello Bahiya, actually, by mistake, I deleted index.html while I was trying to delete the GitHub-readme-activity-graph submodule file. I tried to delete it, but it is not opening in GitHub I can delete it in my vs code. Can you Please guide me and please check whether I made a blunder or not? I have been using GitHub for the last few days, so I have less knowledge regarding all this. And apologise for my mistake once again

**Comment by Kshitij978** [comment]
> > @AbhishekTiwari23 everything looks great in the screen recording ✨
> > Please add these changes too starting from here: [#151 (comment)](https://github.com/Ashutosh00710/github-readme-activity-graph/pull/151#discussion_r987151544) to here: [#151 (comment)](https://github.com/Ashutosh00710/github-readme-activity-graph/pull/151#discussion_r987661842)
> > Also remove [this folder](https://github.com/Ashutosh00710/github-readme-activity-graph/pull/151/files#diff-bb6a6a9d2a3f8829c84b5a3d99e120f8334975e802e7096466045bf28e6e9910)
> > You can commit after doing all these changes. 👍
> > P.S. Let's call each other by name. We both are learners. 😄
> 
> Hello Bahiya, actually, by mistake, I deleted index.html while I was trying to delete the GitHub-readme-activity-graph submodule file. I tried to delete it, but it is not opening in GitHub I can delete it in my vs code. Can you Please guide me and please check whether I made a blunder or not? I have been using GitHub for the last few days, so I have less knowledge regarding all this. And apologise for my mistake once again


@AbhishekTiwari23  Changes are looking great now 🎉.  Coming to the submodule you can:

**##Solution1**
- Try deleting the submodule from your local system then commit and make PR.

**##Solution2**
_**If the above solution doesn't work then please do the following:**_

1. Save a copy of these changes till now (`index.html`, `style.css`) at some different location on your system
2. Delete the clone of this repo from your system after backing up the changes
3. Delete your fork of this Repo from your GitHub
4. Start fresh by fork and clone
5. Include your changes to the freshly cloned repo. 

NOTE: **If you have to apply Solution2 then please _close this PR_ and create a fresh PR and mention the reference to this PR & a reference to the original issue in your _NEW PR_**


P.S. It's all fine don't worry you didn't break anything. You will get everything with time and practice ✨ You can watch tutorials and read articles for best practices. Keep grinding 💪

**Pull request closed without merging**